.github/ISSUE_TEMPLATE/testplan.md

```diff
@@ -983,6 +983,13 @@ tsh bench sessions --max=5000 --web user ls
 - [ ] Verify that all 5000 sessions are able to be established.
 - [ ] Verify that tsh and the web UI are still functional.
 
+### Robustness
+
+* Connectivity Issues:
+
+- [ ] Verify that a lack of connectivity to Auth does not prevent access to resources which do not require a moderated session from an already issued certificate
```

Comment:
We should also add that this requires async recording mode, since sync recording mode requires connection to auth.